packages/shopping-cart/src/shopping-cart-actions.ts

```diff
@@ -0,0 +1,56 @@
+import { createRequestCartProducts } from './create-request-cart-product';
+import type {
+	AddProductsToCart,
+	ReplaceProductsInCart,
+	RemoveProductFromCart,
+	ReplaceProductInCart,
+	UpdateTaxLocationInCart,
+	ApplyCouponToCart,
+	RemoveCouponFromCart,
+	ReloadCartFromServer,
+	RequestCartProduct,
+	DispatchAndWaitForValid,
+	ShoppingCartActionCreators,
+} from './types';
+
+export function createActionCreators(
```

Comment:
Good point. I'm going to keep calling them `actions`, not because they are dispatch/reducer actions, but because they are "actions you can take on the shopping cart".